README.md

```diff
@@ -42,7 +44,11 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+foods.forEach(function(fruits) {
+    fruits.forEach(function(fruit) {
+        console.log("I want to eat a " + fruit);
+    });
+});
```

Comment:
Thanks, I totally missed the clear detail that pizza, fries or hamburger are not fruits and that it wasn't asking to print out all foods.  Working too quickly I guess :)